### PR TITLE
Djid support

### PR DESCRIPTION
**Pull request by zefciu**
* Javascript files for djid added
* require.js used

### CODE REVIEW

File: src/ralph/ui/templates/ui/base.html [hunk 1]
```diff
@@ -13,26 +13,60 @@
...
+                            'deps': [
+                                'jquery',
+                                'jquery-ui'
+                            ]
+                        },
+                    }
                 });
             </script>
-            <script type="text/javascript" src="{{ STATIC_URL }}menu.js">
+            <!-- <script type="text/javascript" src="{{ STATIC_URL }}menu.js"> -->
```
Comment (xliiv) [inline src/ralph/ui/templates/ui/base.html]: ?

File: src/ralph/ui/static/ui/main.js [hunk 2]
```diff
@@ -1,284 +1,295 @@
...
     });
-    $('.select-toggle').click(function() {
-        $('input[name="select"]').each(function () {
-            this.checked = !this.checked;
+    
+    
+    $(function ($) {
+        bob.initialize();
+        menu.initialize();
+        bob_ajax_select.initialize()
```
Comment (xliiv) [inline src/ralph/ui/static/ui/main.js]: missing semi colon

**Comment by xliiv** [comment]
:ok:

**Comment by ar4s** [comment]
``src/ralph/ui/static/jquery-ui/external/jquery/jquery.js`` is for what?

File: src/ralph/ui/static/ui/main.js [hunk 3]
```diff
@@ -1,284 +1,295 @@
...
-        t.show();
+        $('form.search-form').submit(function () {
+            var $form = $(this)
+            var fields = $form.find('input[value!=""],textarea,select').serialize();
+            var action = $form.attr('action') || '';
+            window.location = action + '?' + fields;
+            return false;
+        });
+        $('.close').click(function () {
+            if ($(this).attr('data-dismiss') == 'alert'){
```
Comment (kula1922) [inline src/ralph/ui/static/ui/main.js]: what if not alert? In my opinion everything with class close should be clickable and do some actions.

**Comment by vi4m** [comment]
Szymon, can you finally rename djid, to something more natural?

File: src/ralph/ui/static/ui/main.js (hunk 3 above)
Comment (zefciu) [inline src/ralph/ui/static/ui/main.js]: Please bother Wojtek Matyskiewicz with this.

**Comment by ar4s** [comment]
:ok:

**Pull request merged**